recipes/oneloop/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "oneloop" %}
+{% set version = "3.7.1" %}
+
+package:
+  name: OneLOop
```

Comment:
Hardcoding this instead of using

```yaml
name: {{ name }}
```

to allow for the package's strange preferred typesetting to show up on https://anaconda.org/conda-forge/oneloop in the future.

Reply:
Oh, the linter doesn't like that. Never mind.